easybuild/easyconfigs/p/pkgconf/pkgconf-2.2.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'pkgconf'
+version = '2.2.0'
+
+homepage = 'https://github.com/pkgconf/pkgconf'
+
+description = """pkgconf is a program which helps to configure compiler and linker flags for development libraries.
+ It is similar to pkg-config from freedesktop.org."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://distfiles.ariadne.space/pkgconf/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['28f8dfc279a10ef66148befa3f6eb266e5f3570316600208ed50e9781c7269d8']
+
+# add pkgconfig directories in the system to list of default search paths
+preconfigopts = 'EB_SYS_PC_PATH=":$(find /usr -xdev -type d -name "pkgconfig" -printf %p: 2>/dev/null)";'
+configopts = '--with-pkg-config-dir="%(installdir)s/lib/pkgconfig:%(installdir)s/share/pkgconfig${EB_SYS_PC_PATH%:}"'
```

Comment:
is there any related issue to this change?

Reply:
my bad, this is the easyconfig for `SYSTEM`, so it is expected to also consider the paths in the host system.